cypress/integration/address_book.spec.js

```diff
@@ -0,0 +1,65 @@
+import 'cypress-file-upload';
+const path = require("path");
+
+const NAME = "Owner1"
+const EDITED_NAME = "Edited Owner1"
+const ADDRESS = "0x61a0c717d18232711bC788F19C9Cd56a43cc8872"
+const ENS_NAME = "francotest.eth"
+const RINKEBY_TEST_SAFE = "rin:0xFfDC1BcdeC18b1196e7FA04246295DE3A17972Ac"
+const GNO_TEST_SAFE = "gno:0xB8d760a90a5ed54D3c2b3EFC231277e99188642A"
+const RINKEBY_CSV_ENTRY = {
+    name:'rinkeby user 1',
+    address:'0x730F87dA2A3C6721e2196DFB990759e9bdfc5083'
+}
+const GNO_CSV_ENTRY = {
+    name: 'gno user 1',
+    address:'0x61a0c717d18232711bC788F19C9Cd56a43cc8872'
+}
+
+describe('Address_book', () => {
+    it('Address book entries add/removal', () => {
+        cy.visit(`/${RINKEBY_TEST_SAFE}/address-book`)
+
+        cy.findByText('Create entry', {timeout: 8000}).click()
```

Comment:
8 seconds? Wow.

Reply:
The page is just loading here and it might take bit, so this is giving time to find the button. If the button shows up in less than 8 seconds it clicks it right away.